packages/template-retail-react-app/app/pages/cart/index.test.js

```diff
@@ -187,20 +189,11 @@ describe('Empty cart tests', function () {
 })
 
 describe('Rendering tests', function () {
-    test('Renders skeleton before rendering cart items, shipping info', async () => {
+    test('Renders skeleton initially', async () => {
         renderWithProviders(<Cart />)
-        await waitFor(() => {
-            expect(screen.getByTestId('sf-cart-skeleton')).toBeInTheDocument()
-            expect(screen.queryByTestId('sf-cart-container')).not.toBeInTheDocument()
-        })
-        await waitFor(async () => {
-            expect(screen.getByTestId('sf-cart-container')).toBeInTheDocument()
-            expect(screen.getByText(/Belted Cardigan With Studs/i)).toBeInTheDocument()
-        })
-        const summary = screen.getByTestId('sf-order-summary')
-        expect(await within(summary).findByText(/promotion applied/i)).toBeInTheDocument()
-        expect(within(summary).getByText(/free/i)).toBeInTheDocument()
-        expect(within(summary).getAllByText(/61.43/i).length).toEqual(2)
+
+        expect(screen.getByTestId('sf-cart-skeleton')).toBeInTheDocument()
+        expect(screen.queryByTestId('sf-cart-container')).not.toBeInTheDocument()
```

Comment:
Simplified this test by checking for the skeletons only. Checking for the other things has already been done by the other tests in this file.